cmd/common/image/build.go

```diff
@@ -110,6 +111,7 @@ func NewBuildCmd() *cobra.Command {
 	cmd.Flags().StringVarP(&opts.outputDir, "output", "o", "", "Path to a folder to store generated files while building")
 	cmd.Flags().StringVarP(&opts.image, "tag", "t", "", "Name for the built image (format name:tag)")
 	cmd.Flags().StringVar(&opts.builderImage, "builder-image", builderImage, "Builder image to use")
+	cmd.Flags().StringVar(&opts.cflags, "cflags", "", "Additional flags to pass to the ebpf compiler")
```

Comment:
How does it play with cflags defined in build.yaml? Are they overwritten? If that's the case, we'd need to document it.

Reply:
Good catch.
I updated it so the `cflags` flags from the `ig image build --cflags` command get appended to the one from the `build.yaml`